applications/crossbar/priv/couchdb/schemas/storage.attachment.google_drive.json

```diff
@@ -11,21 +11,8 @@
             "type": "string"
         },
         "settings": {
-            "description": "Settings for the Google Drive account",
-            "properties": {
-                "folder_id": {
-                    "description": "Folder ID in which to store the file, if any",
-                    "type": "string"
-                },
-                "oauth_doc_id": {
-                    "description": "Doc ID in the system 'auth' database",
-                    "type": "string"
-                }
-            },
-            "required": [
-                "oauth_doc_id"
-            ],
-            "type": "object"
+            "$ref": "storage.attachment.settings",
```

Comment:
why are you creating a $ref here ? the oauth_doc_id is missing the kazoo-validation